validation/nsa/internal/rollup-debug.config.js

```diff
@@ -0,0 +1,14 @@
+import { nodeResolve } from "@rollup/plugin-node-resolve";
```

Comment:
The whole internal directory should really be moved to a separate package as it causes a lot of clutter and "intimidating" code/configuration. Besides a bugfix/improvement can then be easily done by updating the version.